plugins/gatsby-source-jenkinsplugins/utils.js

```diff
@@ -159,6 +161,26 @@ const fetchPluginData = async ({createNode, reporter}) => {
         page = pluginsContainer.page + 1;
     } while (!page || pluginsContainer.page < pluginsContainer.pages);
     await Promise.all(promises);
+    const updateUrl = 'https://updates.jenkins.io/current/update-center.actual.json';
+    const updateData = await requestGET({url: updateUrl, reporter});
+    const suspendedPromises = [];
+    for (const deprecation of Object.keys(updateData.deprecations)) {
```

Comment:
This does not actually list plugins whose distribution is suspended, but those marked deprecated -- this feature exists just so that plugins can be marked deprecated while being suspended, but having the customizable URL is a benefit that could result in plugins being marked deprecated through this feature even while still being distributed.

Some plugins have been removed without having an entry here (e.g. `emotional-hudson`). Other plugins may have an entry here without being suspended (unlikely, but as I wrote, possible as it has a benefit).

At a minimum, this needs to cross-check that no plugin found here is still actively distributed before just showing the placeholder page. Then we can just accept that _some_ (few) plugins still don't show up.

The alternative is parsing https://github.com/jenkins-infra/update-center2/blob/master/resources/artifact-ignores.properties and relying on ugly internals of that tool, or extending it to list what's suspended. As a start, the current approach is pretty reasonable if we ensure no currently distributed plugin would have a placeholder page.

Reply:
Great, then this is good to go from my POV :+1:

Reply:
plugins with more than 100 installs (as of April 1) that don't have a deprecation entry:
```
7205 copy-to-slave
5930 build-flow-plugin
4691 tfs
3093 dynamicparameter
1629 perforce
1276 svn-tag
1047 kubernetes-pipeline-steps
761 scripttrigger
595 grails
559 kubernetes-pipeline-aggregator
558 kubernetes-pipeline-arquillian-steps
435 kubernetes-ci
367 zap-pipeline
333 persona
278 jshint-checkstyle
230 xtrigger
229 play-autotest-plugin
200 pipeline-classpath
188 puppet-enterprise-pipeline
184 simple-travis-runner
181 groovyaxis
162 gcm-notification
152 cvs-tag
148 fortify360
101 jslint-checkstyle
```